<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloorWindow</class>
 <widget class="QMainWindow" name="FloorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="Equipment">
      <property name="geometry">
       <rect>
        <x>703</x>
        <y>320</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Save">
      <property name="geometry">
       <rect>
        <x>703</x>
        <y>464</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Next_floor">
      <property name="geometry">
       <rect>
        <x>703</x>
        <y>176</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="StartBattle">
      <property name="geometry">
       <rect>
        <x>277</x>
        <y>176</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="floor_title">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>32</y>
        <width>721</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Choose.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Item">
      <property name="geometry">
       <rect>
        <x>277</x>
        <y>464</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Status">
      <property name="geometry">
       <rect>
        <x>277</x>
        <y>320</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Quit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>608</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="list_page">
     <widget class="QListWidget" name="item_list">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>640</width>
        <height>360</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="use_item">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>520</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="back_item">
      <property name="geometry">
       <rect>
        <x>723</x>
        <y>520</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="item_name">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="item_description">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="status_page">
     <widget class="QLabel" name="knight">
      <property name="geometry">
       <rect>
        <x>-290</x>
        <y>0</y>
        <width>1100</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/knight.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Total_floors_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Total floors.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Current_floor_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Current floor.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Name_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Name.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Level_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>180</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/Level.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="EXP">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/exp.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Funds_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/funds.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="HP">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>330</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/HP.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="MP">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/MP.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="AP">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>430</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/AP.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ATK_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>480</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/ATK.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DEF_title">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>530</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resources/images/DEF.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="back_status">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>600</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="ExpBar">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>230</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="HPBar">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>330</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="MPBar">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>380</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="APBar">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>430</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Level">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>180</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ATK">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>480</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>ATK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Total_floors">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>80</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>Total Floors</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DEF">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>530</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>DEF</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Funds">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>280</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>Funds</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Current_floor">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>130</y>
        <width>301</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gold;</string>
      </property>
      <property name="text">
       <string>Current Floor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="FloorDialog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="FloorDialog_box">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>640</width>
        <height>360</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 2px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QListWidget" name="FloorDialog_list">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>601</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="FloorDialog_next">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>470</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
